Warmup-1/backAround/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export function backAround(myString: string): string {
+  const stringArray = [...myString];
+  const myChar = stringArray[stringArray.length - 1];
```

Comment:
Naming seems a bit difficult to understand what you are doing here. What would you rename it to to make it easier for another developer to understand?